From 32ee37145e68cf8194e6cc73cfb8832be8567616 Mon Sep 17 00:00:00 2001
From: Tom White <tom@cloudera.com>
Date: Fri, 2 Dec 2011 17:46:52 -0800
Subject: [PATCH 010/795] MR1: CLOUDERA-BUILD. Remove smoke tests.
 (cherry picked from commit 89d156069c0e8d57a868215d8184519666f53b05)

(cherry picked from commit e6a2da786ccc6d2babec921a64495bade090c9f6)
---
 build.xml |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/build.xml b/build.xml
index 8b3f44e..d89ee91 100644
--- a/build.xml
+++ b/build.xml
@@ -710,7 +710,6 @@
   <patternset id="empty.exclude.list.id" />
     <patternset id="commit.smoke.exclude.list.id">
     <excludesfile name="${test.commit.tests.file}"/>
-    <excludesfile name="${test.smoke.tests.file}"/>
   </patternset>
 
   <!-- ================================================================== -->
@@ -800,7 +799,7 @@
     </sequential>
   </macrodef>
 
-  <target name="test-core" depends="test-commit, test-smoke,
+  <target name="test-core" depends="test-commit, 
     test-core-excluding-commit-and-smoke,
     test-core-all-withtestcaseonly, jar-test"
     description="Run core unit tests">
-- 
1.7.0.4

